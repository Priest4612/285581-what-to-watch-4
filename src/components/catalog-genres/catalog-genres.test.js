import React from 'react';
import renderer from 'react-test-renderer';

import CatalogGenres from './catalog-genres';
import genresMock from '../../mocks/test-genres-mock.json';


it(`Render CatalogGenres`, () => {
  const tree = renderer
    .create(<CatalogGenres
      genres={genresMock}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
