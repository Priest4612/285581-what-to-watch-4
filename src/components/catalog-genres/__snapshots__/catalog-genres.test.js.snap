// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render CatalogGenres 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item catalog__genres-item--active"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Все жанры
    </a>
  </li>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Комедия
    </a>
  </li>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Музыка
    </a>
  </li>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Научный
    </a>
  </li>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Мюзикл
    </a>
  </li>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Реалити-шоу
    </a>
  </li>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Кинокомикс
    </a>
  </li>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Вестерн
    </a>
  </li>
  <li
    className="catalog__genres-item false"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Ток-шоу
    </a>
  </li>
</ul>
`;
